cmd/server-mux.go

```diff
@@ -441,6 +442,8 @@ func (m *ServerMux) ListenAndServe(certFile, keyFile string) (err error) {
 				RawQuery: r.URL.RawQuery,
 				Fragment: r.URL.Fragment,
 			}
+			lmsg := fmt.Sprintf("redirect for %s", u.String())
```

Comment:
Well, I was only changing the default logrus level in hopes of making it configurable later. I tried that earlier but was unable to make it work.

Our QA organization relies on simple local logs and is not interested in setting up a roundabout notification system, so they will probably continue building with my fork or move back to fake s3 at some point. Feel free to close this request if it is unwanted.